stories/tokens.stories.js

```diff
@@ -0,0 +1,345 @@
+/**
+ * Color Tokens stories.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { Cell, Grid, Row } from '../assets/js/material-components';
+import { Fragment } from '@wordpress/element';
+
+import './tokens.scss';
+
+function TokenRow( { children } ) {
+	return <div className="googlesitekit-color-token-row">{ children }</div>;
+}
+function TokenColumn( { children } ) {
+	return <div className="googlesitekit-color-token-column">{ children }</div>;
+}
+
+function Token( { name, label, reference } ) {
+	return (
+		<div
+			className={ `googlesitekit-color-token googlesitekit-color-token--${ name }` }
+		>
+			<p className="googlesitekit-color-token--label">{ label }</p>
+			<p className="googlesitekit-color-token--reference">
+				{ reference }
+			</p>
+		</div>
+	);
+}
+
+const SurfacesTokens = () => (
+	<Fragment>
+		<h2>Surfaces</h2>
+		<TokenRow>
+			<Token
+				name="surfaces-on-background"
+				label="On Background"
+				reference="N-900"
+			/>
+			<Token
+				name="surfaces-on-background-variant"
+				label="On Background Variant"
+				reference="N-500"
+			/>
+		</TokenRow>
+		<Token name="surfaces-background" label="Background" reference="N-10" />
+		<Token
+			name="surfaces-on-surface"
+			label="On Surface"
+			reference="N-900"
+		/>
+		<Token
+			name="surfaces-on-surface-variant"
+			label="On Surface Variant"
+			reference="N-500"
+		/>
+		<Token name="surfaces-surface" label="Surface (0)" reference="N-0" />
+		<Token name="surfaces-surface-1" label="Surface 1" reference="N-50" />
+		<Token name="surfaces-surface-2" label="Surface 2" reference="N-100" />
+		<TokenRow>
+			<Token
+				name="surfaces-inverse-on-surface"
+				label="Inverse On Surface"
+				reference="N-900"
```

Comment:
This should be `N-50`:

```suggestion
				reference="N-50"
```